element_array_ephys/ephys_organoids.py

```diff
@@ -128,382 +129,180 @@ class AcquisitionSoftware(dj.Lookup):
-        phi (decimal (5, 2) ): azimuth - rotation about the dv-axis in degrees relative to the positive x-axis.
-    """
-
+class RawData(dj.Imported):
     definition = """
-    # Brain Location of a given probe insertion.
-    -> ProbeInsertion
-    ---
-    -> SkullReference
-    ap_location: decimal(6, 2) # (um) anterior-posterior; ref is 0; more anterior is more positive
-    ml_location: decimal(6, 2) # (um) medial axis; ref is 0 ; more right is more positive
-    depth:       decimal(6, 2) # (um) manipulator depth relative to surface of the brain (0); more ventral is more negative
-    theta=null:  decimal(5, 2) # (deg) - elevation - rotation about the ml-axis [0, 180] - w.r.t the z+ axis
-    phi=null:    decimal(5, 2) # (deg) - azimuth - rotation about the dv-axis [0, 360] - w.r.t the x+ axis
-    beta=null:   decimal(5, 2) # (deg) rotation about the shank of the probe [-180, 180] - clockwise is increasing in degree - 0 is the probe-front facing anterior
-    """
-
-
-@schema
-class EphysRecording(dj.Imported):
-    """Automated table with electrophysiology recording information for each probe inserted during an experimental session.
-
-    Attributes:
-        ProbeInsertion (foreign key): ProbeInsertion primary key.
-        probe.ElectrodeConfig (dict): probe.ElectrodeConfig primary key.
-        AcquisitionSoftware (dict): AcquisitionSoftware primary key.
-        sampling_rate (float): sampling rate of the recording in Hertz (Hz).
-        recording_datetime (datetime): datetime of the recording from this probe.
-        recording_duration (float): duration of the entire recording from this probe in seconds.
-    """
-
-    definition = """
-    # Ephys recording from a probe insertion for a given session.
-    -> Session
-    -> ProbeInsertion
+    -> Subject
+    start_time        : datetime # date and time of file creation
     ---
-    -> probe.ElectrodeConfig
-    -> AcquisitionSoftware
-    sampling_rate: float # (Hz)
-    recording_datetime: datetime # datetime of the recording from this probe
-    recording_duration: float # (seconds) duration of the recording from this probe
+    file_name         : varchar(32) # name of the file
+    file_path         : filepath@external-raw  
     """
 
-    class EphysFile(dj.Part):
-        """Paths of electrophysiology recording files for each insertion.
-
-        Attributes:
-            EphysRecording (foreign key): EphysRecording primary key.
-            file_path (varchar(255) ): relative file path for electrophysiology recording.
-        """
-
-        definition = """
-        # Paths of files of a given EphysRecording round.
-        -> master
-        file_path: varchar(255)  # filepath relative to root data directory
-        """
-
     def make(self, key):
-        """Populates table with electrophysiology recording information."""
         session_dir = find_full_path(
             get_ephys_root_data_dir(), get_session_directory(key)
         )
```

Comment:
```suggestion
        subject_dir = find_full_path(
            get_ephys_root_data_dir(), get_subject_directory(key)
        )
```

Reply:
For clarity, let's rename all instances of `session_directory` to `subject_directory`.